### DISCUSSION TITLE
How does OmniBOR benefit folks

### DISCUSSION THREAD

**Discussion by dpp** [omnibor/spec#49] [General]
OmniBOR allows the creation of a lineage graph for any piece of software.

What are the benefits of this lineage graph and for whom?

This is a discussion to enumerate the different constituencies (developers, security folks, executives, open source communities, etc.).

Once we've enumerated the constituencies, we then discuss benefits to that community and in what context (e.g., development time, production time, incident response time, etc.)

Finally, what ecosystem augmentation (e.g., a public, indexed database of all gitoids and what they relate to) are needed to actually realize the benefits of OmniBOR.

**Comment by alilleybrinker** [comment]
Right, this sounds like you're wanting a Use Cases document, something like [this](https://ntia.gov/sites/default/files/publications/ntia_sbom_use_cases_roles_benefits-nov2019_0.pdf).

**Comment by edwarnicke** [comment]
@alilleybrinker  That link is broken.

**Reply by alilleybrinker** [reply]
Apologies! It's fixed now.

**Comment by alilleybrinker** [comment]
We have a [working draft document for this](https://docs.google.com/document/d/1SUILYVki1JcuSUWXcfHpD1McGQ5lvRze24ipPJ-QXPg/edit?usp=sharing).
